webapp/web/configuration_form.css

```diff
@@ -0,0 +1,65 @@
+.configuration-form {
+    display: grid;
+    grid-template-rows: auto;
+    grid-gap: 20px;
+    border: 1px solid #ddd;
+    border-radius: .5rem;
+    padding: 20px;
+}
+
+.single-form-group {
+    display: flex;
+    padding: 27px 0px 27px 10px;
+    border-radius: .5rem;
+    box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.22);
+}
+
+.single-form-group--in-multi {
```

Comment:
I wonder if we would have more control over the formatting if we do this the other way around - instead of clearing out the styles via a class, we can explicitly set the class for which we want different formatting. For example, you could have a `.single-form-group--shadow`, and then the code in `_buildForm` would look like this:

```
      ..append(
        _singleFormGroup('Automated translations enabled', FormGroupTypes.CHECKBOX)
          ..classes.add('single-form-group--shadow')
      )
```

I also think that for the multi-groups (e.g. user and language) we need an indication of the group too, otherwise the label for the multi-form-group appears slightly out of place, and it's not clear what's it's bounding box. So maybe the shadow+border should appear only around the outer groups, whether they're single or multi? (ie. we should also have a `.multi-form-group--shadow`). But where the shadow is is more a design issue, for which we'll need the UI structure above anyway.

Reply:
I think I'd like to do this in the new PR

Reply:
sure, sounds like a good idea, thanks!